Makefile

```diff
@@ -17,6 +17,11 @@ export GO111MODULE = on
 # process build tags
 
 build_tags = netgo
+
+ifeq ($(EXPERIMENTAL),true)
+	build_tags += stable
```

Comment:
@anilCSE The logic is reversed here

maybe the build_tag should also be called `experimental`?